src/drive/web/modules/filelist/FileListHeader.jsx

```diff
@@ -124,12 +135,43 @@ class MobileHeader extends Component {
             onSort={(attr, order) => onFolderSort(folderId, attr, order)}
           />
         )}
+        <div
+          className={classNames(
+            styles['fil-content-mobile-header'],
+            styles['fil-content-header-action']
```

Comment:
We now try to not write CSS in our apps (I know... Drive is full of written CSS...) since cozy-ui comes with a lot of utilities classes. 

For instance, you can use : 
- className={'u-flex'} to have a display: flex
- u-w-auto to have with auto 
- u-p-0  to have padding 0 

You can find all these utilities here: https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html